src/status_im2/contexts/wallet/collectible/view.cljs

```diff
@@ -82,6 +79,21 @@
        :num-columns             2
        :content-container-style style/traits-container}]]))
 
+(defn activity-item
+  [item]
+  [:<>
+   [quo/divider-date (:timestamp item)]
+   [quo/wallet-activity
+    (merge {:on-press #(js/alert "Item pressed")}
```

Comment:
If I remember well, in a recent PR @J-Son89  said we should avoid adding these alerts, and treat the screens as final versions.

am I right @J-Son89 ?

Reply:
if i recall correctly, I was referring to a quo component :)

Reply:
Oh, ok @vkjr please ignore my comment, thanks!.

Reply:
Yeah, I will remove one, it doesn't do anything valuable